### ISSUE TITLE
Sonoff ZBDongle-E / Firmware 7.4.1 / docker unable to start

### ISSUE THREAD

**Issue by fred-gb** [Koenkk/zigbee2mqtt#22228]
### What happened?

Hello,
Unable to start zigbee2mqtt after upgrade.

### What did you expect to happen?

Following this issue: https://github.com/Koenkk/zigbee2mqtt/issues/21366

Start with docker like before

### How to reproduce it (minimal and precise)

I upgrade with with tutorial: https://github.com/Koenkk/zigbee2mqtt/discussions/21462

My dongle is update to 7.4.1.

I change to ember driver:

```
serial:
  port: /dev/ttyACM0
  adapter: ember
frontend:
  port: 8080
  host: 0.0.0.0
  auth_token: K01XRFE:VGnYGauc-EUL2f7fmDIggg-U
  url: http://zigbee2mqtt.service.lan
advanced:
  log_level: debug
```

 

### Zigbee2MQTT version

1.36.1

### Adapter firmware version

Unable to access

### Adapter

Sonoff ZBDongle-E

### Setup

LXC container Ubuntu, Nomad as orchestrator, Z2M Docker version

### Debug log

```


Using '/app/data' as data directory
Zigbee2MQTT:debug 2024-04-16 18:33:12: Loaded state from file /app/data/state.json
Zigbee2MQTT:info  2024-04-16 18:33:12: Logging to console and directory: '/app/data/log/2024-04-16.18-33-12' filename: log.txt
Zigbee2MQTT:debug 2024-04-16 18:33:12: Removing old log directory '/app/data/log/2024-04-16.18-06-09'
Zigbee2MQTT:info  2024-04-16 18:33:12: Starting Zigbee2MQTT version 1.36.1 (commit #ffc2ff1)
Zigbee2MQTT:info  2024-04-16 18:33:12: Starting zigbee-herdsman (0.40.3)
Zigbee2MQTT:debug 2024-04-16 18:33:12: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null,"delay":null,"disableLED":false},"backupPath":"/app/data/coordinator_backup.json","databaseBackupPath":"/app/data/database.db.backup","databasePath":"/app/data/database.db","network":{"channelList":[11],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6754},"serialPort":{"adapter":"ember","path":"/dev/ttyACM0"}}'
======== Ember Adapter Starting ========
======== EZSP starting ========
======== ASH NCP reset ========
======== ASH starting ========
======== ASH NCP reset ========
======== ASH starting ========
======== ASH NCP reset ========
======== ASH starting ========
======== ASH NCP reset ========
======== ASH starting ========
======== ASH NCP reset ========
======== ASH starting ========
Zigbee2MQTT:error 2024-04-16 18:33:25: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2024-04-16 18:33:25: Failed to start zigbee
Zigbee2MQTT:error 2024-04-16 18:33:25: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2024-04-16 18:33:25: Exiting...
Zigbee2MQTT:error 2024-04-16 18:33:25: Error: Failed to start EZSP layer with status=HOST_FATAL_ERROR.
    at EmberAdapter.initEzsp (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/emberAdapter.ts:804:19)
    at EmberAdapter.start (/app/node_modules/zigbee-herdsman/src/adapter/ember/adapter/emberAdapter.ts:2661:24)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:132:29)
    at Zigbee.start (/app/lib/zigbee.ts:62:27)
    at Controller.start (/app/lib/controller.ts:109:27)
    at start (/app/index.js:107:5)
Using '/app/data' as data directory
Zigbee2MQTT:debug 2024-04-16 18:33:38: Loaded state from file /app/data/state.json
Zigbee2MQTT:info  2024-04-16 18:33:38: Logging to console and directory: '/app/data/log/2024-04-16.18-33-38' filename: log.txt
Zigbee2MQTT:debug 2024-04-16 18:33:38: Removing old log directory '/app/data/log/2024-04-16.18-15-33'
Zigbee2MQTT:info  2024-04-16 18:33:38: Starting Zigbee2MQTT version 1.36.1 (commit #ffc2ff1)
Zigbee2MQTT:info  2024-04-16 18:33:38: Starting zigbee-herdsman (0.40.3)
Zigbee2MQTT:debug 2024-04-16 18:33:38: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null,"delay":null,"disableLED":false},"backupPath":"/app/data/coordinator_backup.json","databaseBackupPath":"/app/data/database.db.backup","databasePath":"/app/data/database.db","network":{"channelList":[11],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6754},"serialPort":{"adapter":"ember","path":"/dev/ttyACM0"}}'
======== Ember Adapter Starting ========
======== EZSP starting ========
======== ASH NCP reset ========
======== ASH starting ========
======== ASH NCP reset ========
======== ASH starting ========
Zigbee2MQTT:debug 2024-04-16 18:33:43: Saving state to file /app/data/state.json
Zigbee2MQTT:error 2024-04-16 18:33:43: Not connected to MQTT server!
Zigbee2MQTT:error 2024-04-16 18:33:43: Cannot send message: topic: 'zigbee2mqtt/bridge/state', payload: 'offline
Zigbee2MQTT:info  2024-04-16 18:33:43: Disconnecting from MQTT server

```

```
test -w /dev/ttyACM0 && echo success || echo failure
success
```

**Comment by fred-gb** [comment]
Not solved, but I can downgrade to 7.3 to I found a normal mode, but I can use my device PTM216. I need to retry upgrade. But now, I have this additonals informations:

```
Version de Zigbee2MQTT
    [1.36.1](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.36.1) commit: [ffc2ff1](https://github.com/Koenkk/zigbee2mqtt/commit/ffc2ff1)

Type de coordinateur
    EZSP v12

Révision du coordinateur
    7.3.1.0 build 176

Adresse IEEE du Coordinateur
    0xe0798dfffe719490

Version de l'interface
    0.6.161

zigbee_herdsman_converters_version
    19.11.2

zigbee_herdsman_version
    0.40.3
```

Thanks

**Comment by fred-gb** [comment]
I bought a second dongle Sonoff ZBDongle-E, and firmware upgrade works. And connection with zigbee2mqtt works.

I don't know why on previous dongle it does not working.

Thanks
